cves/CVE-2018-13304.yml

```diff
@@ -64,7 +64,23 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When encoding an 8-bit video with the MPEG4 codec, there is a check to ensure
+  the supported MPEG4 profile for 8-bit video, Simple Studio Profile, is being used. 
+  Profiles are basically different specifications aimed at optimizing encoding/decoding
+  for a particular video type, which are then implemented into actual software. 
+  The active profile is tracked by both the MPEG4 encoder and the decoder from 
+  the original video, and both values are checked throughout the system to 
+  ensure only supported profiles are being used. However, the decoding source 
+  can change throughout the encoding process. The AVI video format consists
+  of "chunks" of video strung together. These chunks can actually
+  be using completely different compression codecs, including MPEG4. 
+  So using an AVI file that has an MPEG4 chunk that could be 
+  decoded with a different profile could cause the profile check to fail, 
+  causing an error to be thrown and stopping the conversion process.
+  While the decoder attempts to correctly switch the profile back and forth
```

Comment:
Good job describing how encoding is supposed to work, but I would begin by describing the vulnerability first and how it causes an error in the decoding process first. Then describe more about the encoding process. Very detailed overall.